core/src/autogluon/core/models/greedy_ensemble/ensemble_selection.py

```diff
@@ -147,6 +147,8 @@ def _fit(self, predictions, labels, time_limit=None, sample_weight=None):
     def _calculate_regret(self, y_true, y_pred_proba, metric, sample_weight=None):
         if metric.needs_pred or metric.needs_quantile:
             preds = get_pred_from_proba(y_pred_proba=y_pred_proba, problem_type=self.problem_type)
+        elif metric.needs_quantile:
```

Comment:
isn't this `elif` redundant to the `if` above?

Reply:
Yes, it is. I'd removed the redudant part.